loader/loader.c

```diff
@@ -4844,28 +4844,30 @@ VkResult loader_create_device_chain(const VkPhysicalDevice pd, const VkDeviceCre
 
         // If layer debugging is enabled, let's print out the full callstack with layers in their
         // defined order.
-        if ((loader_get_debug_level() & VULKAN_LOADER_LAYER_BIT) != 0) {
-            loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "vkCreateDevice layer callstack setup to:");
-            loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "   <Application>");
-            loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "     ||");
-            loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "   <Loader>");
-            loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "     ||");
-            for (uint32_t cur_layer = 0; cur_layer < num_activated_layers; ++cur_layer) {
-                uint32_t index = num_activated_layers - cur_layer - 1;
-                loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "   %s", activated_layers[index].name);
-                loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "           Type: %s",
-                           activated_layers[index].is_implicit ? "Implicit" : "Explicit");
-                if (activated_layers[index].is_implicit) {
-                    loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "               Disable Env Var:  %s",
-                               activated_layers[index].disable_env);
+        uint32_t layer_driver_bits = VULKAN_LOADER_LAYER_BIT | VULKAN_LOADER_DRIVER_BIT;
+        if ((loader_get_debug_level() & layer_driver_bits) != 0) {
+            loader_log(inst, layer_driver_bits, 0, "vkCreateDevice layer callstack setup to:");
+            loader_log(inst, layer_driver_bits, 0, "   <Application>");
+            loader_log(inst, layer_driver_bits, 0, "     ||");
+            loader_log(inst, layer_driver_bits, 0, "   <Loader>");
+            loader_log(inst, layer_driver_bits, 0, "     ||");
+            if ((loader_get_debug_level() & VULKAN_LOADER_LAYER_BIT) != 0) {
+                for (uint32_t cur_layer = 0; cur_layer < num_activated_layers; ++cur_layer) {
+                    uint32_t index = num_activated_layers - cur_layer - 1;
+                    loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "   %s", activated_layers[index].name);
+                    loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "           Type: %s",
+                               activated_layers[index].is_implicit ? "Implicit" : "Explicit");
+                    if (activated_layers[index].is_implicit) {
+                        loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "               Disable Env Var:  %s",
+                                   activated_layers[index].disable_env);
+                    }
+                    loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "           Manifest: %s", activated_layers[index].manifest);
+                    loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "           Library:  %s", activated_layers[index].library);
+                    loader_log(inst, VULKAN_LOADER_LAYER_BIT, 0, "     ||");
```

Comment:
This change seems a bit odd, since now if you specify DRIVER in the log output you'll get the `<app>--<loader>--<driver>` output but since there isn't any additional info about the device, it doesn't help the user any.

Reply:
There is, just in the next part of the change.  I don't know the driver until we get to the terminator (line 5636)